public/login.html

```diff
@@ -105,7 +105,6 @@
         scale: 1.1
       })
     </script>
-    <script src="js/main.js"></script>
```

Comment:
@notrodes and I thought that input validation is not being used anywhere because the input doesn't seem to be validated at all (`main.js` seems to check for an email format, but non-email usernames are accepted). I haven't been able to trigger the input validation on the production server (https://aspine.cpsd.us). I'm happy that you brought this up because I'm a bit confused about the input validation. If you could provide further insight, that would be great.

Reply:
I agree that main.js doesn't appear to do any useful validation. Removing it seems to be the right thing to do, but maybe we can add an issue for input validation on the login page that would be resolved in a future pull request.

Reply:
The main issue that I see is that the district could change the format of usernames on Aspen at any time and Aspine could simply not work for those people whose usernames are in the new format. While not changing the number of digits, the district has switched the first digit of student IDs and has given some teachers non-standard usernames. It might also be useful to allow any type of username in case we decide to expand later to support teacher accounts (I do know at least one teacher who may be interested).

Reply:
Ok sure that makes sense